### DISCUSSION TITLE
Get RSS Feed by the showUuid or podcastGuid

### DISCUSSION THREAD

**Discussion by Stefan-WW** [skymethod/op3#23] [Q&A]
Hello, 

Is there Any way to get the RSS Feed for an OP3 Podcast by the showUuid or podcastGuid?

i couldn't find any solution in the api documentation.

I would be very grateful for any help or tips
Kind Regards 
Stefan

**Comment by jamescridland** [comment]
GUID to RSS feed... you should use
https://podcastindex-org.github.io/docs-api/#get-/podcasts/byguid

The OP3 showUUID is a proprietary ID, unfortunately.
